simba/utils/enums.py

```diff
@@ -351,7 +354,7 @@ class Defaults(Enum):
     LARGE_MAX_TASK_PER_CHILD = 1000
     CHUNK_SIZE = 1
     SPLASH_TIME = 2500
-    WELCOME_MSG = f'Welcome fellow scientists! \n SimBA v.{pkg_resources.get_distribution("simba-uw-tf-dev").version} \n '
+    WELCOME_MSG = f'Welcome fellow scientists! \n SimBA v.{pkg_resources.get_distribution("simba-uw-tf-dev").version if importlib.util.find_spec("simba-uw-tf-dev") is not None else "dev"} \n '
```

Comment:
For some reason, `importlib.util.find_spec("simba-uw-tf-dev")` returns None in my python3.6 environment. Does something like this work on your end?

```
    SPLASH_TIME = 2500
    try:
        WELCOME_MSG = f'Welcome fellow scientists! \n SimBA v.{pkg_resources.get_distribution("simba-uw-tf-dev").version} \n '
    except pkg_resources.DistributionNotFound:
        WELCOME_MSG = f'Welcome fellow scientists! \n SimBA v. "dev" \n '
    BROWSE_FOLDER_BTN_TEXT = 'Browse Folder'
```

Reply:
yes, i dont like the try except methodology in python but obviously it would work